api/tests/unit/external_resources/test_unit_external_resources_views.py

```diff
@@ -0,0 +1,136 @@
+import simplejson as json
+from django.core.serializers.json import DjangoJSONEncoder
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APIClient
+
+from features.feature_external_resources.models import FeatureExternalResource
+from features.models import Feature
+
+_django_json_encoder_default = DjangoJSONEncoder().default
+
+
```

Comment:
Can we add a unit test that checks that the violating following constraint does not result in 500

```
constraints = [
            models.UniqueConstraint(
                fields=["feature", "url"], name="unique_feature_url_constraint"
            )
        ]
```

Reply:
Test was added